server/pbenchinacan/container-build.sh

```diff
@@ -1,24 +1,29 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
 #
-# This script builds a stand-alone Pbench Server container with just the base
-# software installed.
+# This script builds a container which, when run, starts a Pbench Server.
+#
+# Note:  successfully running this script required adding CAP_SETFCAP (for the
```

Comment:
Along with this very indirect reference to "the other default capabilities" it'd be useful to have a specific list of exactly what setup we need to run this.

(And, yeah, I know you just moved this from the file you deleted; but the "wonderful" thing about moving code is that it forces everyone to re-review it. 😆 )

Reply:
At Peter's prompting, I've deleted the whole `Note`.